analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt

```diff
@@ -53,6 +58,20 @@ class FbAppender : IAnalyticsAppender {
         firebaseAnalytics?.setCurrentScreen(activity, screenName, screenClassOverride)
     }
 
+    override fun trackPageHit(screenClass: String, screenName: String) {
+        if (firebaseAnalytics == null) {
+            LOG.e(LOG_TAG, "trackPageHit failed: firebaseAnalytics is null")
+            return
+        }
+
+        val parameters = mapOf(
```

Comment:
Can't we use `bundleOf` ?

Reply:
we do not have the required imports for that on this module